robot/EDA/resources/SetupPageObject.py

```diff
@@ -13,10 +15,34 @@ def _go_to_page(self):
         self.selenium.switch_window("Home | Salesforce")
         self.salesforce.wait_until_loading_is_complete()
 
+    def click_custom_settings(self, setting):
+        """ Clicks the custom settings using the name of the setting passed from the test
+        """
+        locator = eda_lex_locators["eda_setup"]["custom_settings"].format(setting)
+        self.selenium.wait_until_page_contains_element(locator)
+        self.selenium.wait_until_element_is_visible(locator)
+        self.selenium.click_element(locator)
+
+    def click_custom_settings_action_button(self, button):
+        """ Clicks the custom settings button using the name of the button passed from the test
+        """
+        locator = eda_lex_locators["eda_setup"]["settings_action_button"].format(button)
+        self.selenium.wait_until_page_contains_element(locator)
+        self.selenium.wait_until_element_is_visible(locator)
+        self.selenium.click_element(locator)
+
     def perform_a_quick_find_on(self, item):
         """ Searches for the given item through Quick Find on Setup page """
         self.eda.populate_placeholder("Quick Find", item)
 
+    def select_frame(self, value):
+        """ Selects iframe using the value passed from the test. The value can be iframe id, title
+            etc
+        """
+        locator = eda_lex_locators["eda_setup"]["frame"].format(value)
+        time.sleep(4)
```

Comment:
can u wait for the frame instead of just sleep here

Reply:
@skristem This has been updated and moved to EDA.py as suggested